style-guide.md

```diff
@@ -0,0 +1,38 @@
+## Style Guide
+
+Contributions to this repository should follow the style guide as described in this section.
+
+### Markdown
+
+Markdown files in this project should follow the style enforced by the [markdownlint tool](https://www.npmjs.com/package/markdownlint),
+as configured by the `.markdownlint.yaml` file in the root of the project.
+The `markdownlint` tool can also fix formatting, which can save time with tables in particular.
+
+The following additional rules should be followed but currently are not enforced by tooling:
+
+1. The first mention of a normative reference or an OAS-defined Object in a (sub)*section is a link, additional mentions are not.
+2. OAS-defined Objects such as Schema Objects are written in this style, and are not monospaced.
+3. Use "example" instead of "sample" - this spec is not about statistics.
+4. Use "OpenAPI Object" instead of "root".
+5. Fixed fields are monospaced.
+6. Field values are monospaced in JSON notation: `true`, `false`, `null`, `"header"` (with double-quotes around string values).
+7. A combination of fixed field name with example value uses JS notation: `in: "header"`, combining rules 5 and 6.
+8. An exception to 5-7 is colloquial use, for example "values of type `array` or `object`" - "type" is not monospaced, so the monospaced values aren't enclosed in double quotes.
+9. Use Oxford commas, avoid Shatner commas.
```

Comment:
We should probably provide a link for people who don't get the joke (or the grammar reference):  https://www.latimes.com/archives/blogs/jacket-copy/story/2011-06-30/goodbye-oxford-comma-hello-shatner-comma , https://www.urbandictionary.com/define.php?term=Shatner%20commas, https://elissadigital.wordpress.com/2013/09/16/your-writing-shouldnt-sound-like-william-shatner-or-say-no-to-the-shatner-comma/ should suffice for "Shatner comma" (maybe not urban dictionary- it's ads are sometimes 😬 ) and wikipedia has an [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) explanation.

Reply:
I am not linking to urban dictionary!! But I'll edit (I wasn't familiar with the term myself)

Reply:
OK, links added.